src/exercise/02.md

```diff
@@ -58,9 +58,27 @@ Here's a simple example of the API:
 const elementProps = {id: 'element-id', children: 'Hello world!'}
 const elementType = 'h1'
 const reactElement = React.createElement(elementType, elementProps)
-ReactDOM.render(reactElement, rootElement)
+const root = ReactDOM.createRoot(rootElement)
+root.render(reactElement)
 ```
 
+> 🦉 NOTE: prior to React v18, the API was: `ReactDOM.render` and that's what
+> you'll see in the EpicReact.dev videos. This material has been updated, so
+> you'll want to use the new `React.createRoot` API as demonstrated above.
```

Comment:
This should be ReactDOM.createRoot, right ? @kentcdodds

Reply:
Yes 👍